README.md

```diff
@@ -2,16 +2,14 @@
 
 [![https://www.npmjs.com Package](https://github.com/TIGNUM/pdf-generation-actions-web-component/actions/workflows/npm-publish-package.yml/badge.svg?branch=main&event=workflow_run)](https://github.com/TIGNUM/pdf-generation-actions-web-component/actions/workflows/npm-publish-package.yml)
 
-Print or pdf generation buttons for generating Tignum reports
+Print or generate a pdf from the html passed as string to the Web Component
 
 ## How to load the component
 
-```html
- <actions-report class="style-area" html="<p>Test</p>"></actions-report>
-```
+    <actions-report class="style-area" html="<p>Test</p>"></actions-report>
```

Comment:
You renamed the html prop to htmlString in this PR

Reply:
Breaking change, true, I rolled back. That attribute could create confusion in the future